lib/node.js

```diff
@@ -822,8 +822,102 @@ declare module "vm" {
   // TODO
 }
 
+type zlib$ZlibOptions = {
+  flush: ?number;
+  chunkSize: ?number;
+  windowBits: ?number;
+  level: ?number;
+  memLevel: ?number;
+  strategy: ?number;
+  dictionary: ?Buffer;
+};
```

Comment:
I think it's more accurate to make the keys optional than the values maybe types. I.e., `{ foo?: T }` instead of `{ foo: ?T }`.